test/functional/breadcrumbs.ts

```diff
@@ -131,7 +177,19 @@ suite<Breadcrumbs>('gb-breadcrumbs', ({ flux, mount, stub, itMountsTag }) => {
 export class Model extends BaseModel<Breadcrumbs> {
 
   get queryCrumb() {
-    return this.element(this.html, 'div > .gb-query-crumb');
+    return this.element(this.html, 'div gb-query-crumb');
+  }
+
+  get originalQuery() {
+    return this.element(this.html, 'gb-query-crumb .gb-original-query');
```

Comment:
instead of `this.html` use `this.queryCrumb` as the root element then you can make your CSS selector string shorter